File: go/earth/earth.go
package earth

type Continent struct {
	Name       string
	Area       float64
	Population int
}

func NewContinentList() []Continent {
	Continents := []Continent{
		{Name: "Africa", Area: 32.3e6, Population: 1.216e9},
		{Name: "Antartica", Area: 13e6, Population: 4.0e3},
		{Name: "Asia", Area: 43.3e6, Population: 4.561e9},
		{Name: "Australia", Area: 7.7e6, Population: 25.69e8},
		{Name: "Europa", Area: 10.4e6, Population: 746.4e8},
		{Name: "North America", Area: 24.9e6, Population: 579e8},
		{Name: "South America", Area: 17.8e6, Population: 422.5e8},
	}

	return Continents
}

func Radius() float64 {
	return 6.4e3
}

File: elixir/lib/Earth/earth.ex
# Example of simple module in Elixir
defmodule Earth do
  @radius 6.4e3
  def radius, do: @radius

  def continents do
    [
      %Continent{name: "Africa", area: 30.3e6, population: 1.216e9},
      %Continent{name: "Antartica", area: 13.0e6, population: 4.0e3},
      %Continent{name: "Asia", area: 43.3e6, population: 4.561e9},
      %Continent{name: "Australia", area: 7.7e6, population: 25.69e8},
      %Continent{name: "Europa", area: 10.4e6, population: 746.4e8},
      %Continent{name: "North America", area: 10.4e6, population: 579.0e8},
      %Continent{name: "South America", area: 10.4e6, population: 422.5e8}
    ]
  end
end
